sitemedia/js/components/FilterChoice.js

```diff
@@ -0,0 +1,25 @@
+export default Vue.component ('FilterChoice', {
+    template: `
+    <label :active="active" is="sui-button" role="checkbox" basic circular compact>
+        {{ label }}
+        <input
+            type="checkbox"
+            :value="value"
+            @input="$emit('input', $event.target.value)"
+            :name="name"
+            v-model="active"
+            hidden
+        >
+    </label>
+    `,
+    props: {
+        name: String,
+        label: String,
```

Comment:
"label" is the user-friendly display name; "value" is the `value` attribute that gets used on the `<input>` element. I think if I document the components' props with a docstring like you mentioned this will be clearer.